src/compiler.adoc

```diff
@@ -1073,16 +1073,16 @@ first test fails as it did before, but our leap year test passes as expected.
 
 ==== Async Testing
 
-One of the peculiarities of _ClojureScript_ is that is built up on asynchronous, single
-theaded execution environments, and this has its own challenges.
+One of the peculiarities of _ClojureScript_ is that it runs on asynchronous,
+single-threaded execution environments, and this has its own challenges.
```

Comment:
- clarify: is built up on -> it runs on (clarifying execution environment rather than build environment)
- hyphen: single-threaded (not important really)